docs/id-lifecycle-management/support-systems/partner-management-services/functional-overview/onboard-credential-partner.md

```
@@ -146,7 +146,7 @@ Re-uploading certificate is required in cases when MOSIP Signed Certificate gets
 
 **Step 1: Upload Root Certificate**
 
-Partner admin has to upload the root certificate as the primary steps for any partner onboarding. Please refer the guide here ( [_Keshav Singh_](https://mosip.atlassian.net/wiki/people/712020:db7f76c3-b494-4ddf-93dc-7036472db6d0?ref=confluence) Link to the Root Certificate upload guide under Partner admin end user guide) to know more about the steps.
+Partner admin has to upload the root certificate as the primary steps for any partner onboarding. Please refer the guide [here](partner-administration.md#root-trust-root-ca-certificate-features) to know more about the steps.
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)